bitnami/rabbitmq/templates/_helpers.tpl

```diff
@@ -64,7 +64,7 @@ Get the password secret.
 */}}
 {{- define "rabbitmq.secretPasswordName" -}}
     {{- if .Values.auth.existingPasswordSecret -}}
-        {{- printf "%s" .Values.auth.existingPasswordSecret -}}
+        {{- printf "%s" (include "common.tplvalues.render" (dict "value" .Values.auth.existingPasswordSecret "context" .)) -}}
```

Comment:
Since it's a just a string with the secret name, we can make it simpler and avoid using `common.tplvalues.render`:
```suggestion
        {{- printf "%s" (tpl .Values.auth.existingPasswordSecret $) -}}
```